.gitignore

```diff
@@ -1,6 +1,6 @@
 # editor temporary / backup files
 *~
-
+.venv
```

Comment:
oops. but in fact could we keep it? (I can move to a more sensible location in .gitignore)

Reply:
github being buggy again.. :roll_eyes: